packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -116,30 +116,35 @@ test('visualize NXentry group with implicit default child NXdata group', async (
   expect(tabs[0]).toHaveTextContent(NexusVis.NxSpectrum);
 });
 
-test('show error when encountering malformed NeXus metadata', async () => {
+test('show error when `default` entity is not found', async () => {
   await renderApp();
 
   const errorSpy = mockConsoleMethod('error');
-  await selectExplorerNode('nexus_malformed');
```

Comment:
Note that the `nexus_malformed` entity, which still has `no_signal` as its implicit default child, now shows the "no vis" fallback message too.